libs/net/services/Helpers/KafkaListener`.cs

```diff
@@ -0,0 +1,210 @@
+using System.Text.Json;
+using Confluent.Kafka;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using TNO.Services.Serializers;
+
+namespace TNO.Services;
+
+/// <summary>
+/// KafkaListener class, provides a way to consume messages from Kafka.
+/// </summary>
+/// <typeparam name="TKey"></typeparam>
+/// <typeparam name="TValue"></typeparam>
+public class KafkaListener<TKey, TValue> : IKafkaListener<TKey, TValue>, IDisposable
```

Comment:
Replaced `KafkaListener` with a generic version so that it can handle start/stop actions and failures.